tests/models/test_sasrec.py

```diff
@@ -24,7 +24,34 @@
 
 from .data import DATASET, INTERACTIONS
 
+# pylint: disable=too-many-lines
 
+
+@pytest.mark.parametrize(
```

Comment:
Let's not do this for all of the tests for this model. We are going to write more of them.
Let's do it only for test_u2i.

Reply:
fixed